src/main.ts

```diff
@@ -355,13 +349,10 @@ export default class NewZettel extends Plugin {
             (this.settings.addTitle ? this.settings.separator + title : "") +
             ".md"
           : "";
-      const addAlias = this.settings.addAlias;
       const useLinkAlias = this.settings.useLinkAlias;
       const newLink = (title: string) => {
-        let alias = ''
-        if (addAlias && useLinkAlias) {
-          alias = '|' + title
-        }
+        let alias = useLinkAlias ? `|${title}` : ''
```

Comment:
Good use of a format string I like this move twords this I think its a good direction for us to move in and is cleaner and easier to read.